en/docs/develop/mi-for-vscode/install-wso2-mi-for-vscode.md

```diff
@@ -1,19 +1,19 @@
 # Install Micro Integrator for VS Code
 
-This page walks you through how to get started with the Micro Integrator for VS Code extension by installing it.
+This page walks you through how to get started with the Micro Integrator for VS Code extension.
 
 ## Installation Prerequisites
 
-- Download and install Java SE Development Kit (JDK) version 11 or 17.
+- Download and install Java SE Development Kit (JDK) version 11 or 17. See [tested JDK versions]({{base_path}}/install-and-setup/setup/reference/product-compatibility/#tested-jdks) for more information.
 
-- Set the `JAVA_HOME` environment variable in the system settings.
+- Set the `JAVA_HOME` environment variable in the [system settings]({{base_path}}/install-and-setup/install/installing-mi/#setting-up-java_home).
```

Comment:
No need to add a link here since the same reference is provided in the note immediately below this text.

Reply:
True, but as that is there in a note, added it here too.